Dockerfile

```diff
@@ -1,61 +1,20 @@
-FROM openjdk:8-jre
-LABEL maintainer="dev+jenkins-nvm@dwolla.com"
+FROM dwolla/jenkins-agent-core:debian
 LABEL org.label-schema.vcs-url="https://github.com/Dwolla/jenkins-agent-docker-nvm"
+LABEL maintainer="dev+jenkins-nvm@dwolla.com"
 
-ENV JENKINS_HOME=/home/jenkins \
-    JENKINS_AGENT=/usr/share/jenkins \
-    AGENT_VERSION=2.62.6 \
-    NVM_VERSION=v0.33.6
-
-ENV NVM_DIR="${JENKINS_HOME}/.nvm"
+ENV NVM_VERSION=v0.34.0 \
+    NVM_DIR="${JENKINS_HOME}/.nvm"
 
-WORKDIR ${JENKINS_HOME}
-COPY jenkins-agent install.sh verify.sh /usr/local/bin/
+COPY install.sh verify.sh /usr/local/bin/
 
-# apt-key loop inspired by https://github.com/nodejs/docker-node/issues/340#issuecomment-321669029
+USER root
 RUN set -ex && \
     apt-get update && \
     apt-get install -y \
-        apt-transport-https \
```

Comment:
I am guessing these packages are installed on the `jenkins-agent-core:debian` image?

Reply:
Yep! 

https://github.com/Dwolla/jenkins-agent-docker-core/blob/79af34db214e0c1f58be199bbceb29a374e94ed0/debian/Dockerfile#L18-L28